nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -1276,7 +1276,7 @@ private String csv(Collection<String> values) {
         // create a new hashmap to store the values from the connection
         Map<String, String> map = new HashMap<>();
         responseHttp.headers().names().forEach((key) -> {
```

Comment:
You would want to use StringUtils.isBlank() see other parts of the code.

Reply:
I had used the logic in org.apache.nifi.flowfile.FlowFile.KeyValidator.isValid(String) as a guideline, but I agree that this is a more appropriate solution.